### PR TITLE
token bridge faq

### PR DESCRIPTION
**Pull request by ilariae**
### Description

updated contracts faq

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: build/contract-integrations/faqs.md [hunk 1]
```diff
@@ -28,4 +28,14 @@ Previously referred to as specialized relayers, custom relayers allow you to bui
  - [Relayer Engine](/docs/infrastructure/relayers/run-relayer/#get-started-with-the-relayer-engine){target=\_blank}
  - [Run a Custom Relayer](/docs/infrastructure/relayers/run-relayer/){target=\_blank}
 
-These resources will guide you through building and deploying custom relayers tailored to your use case.
\ No newline at end of file
+These resources will guide you through building and deploying custom relayers tailored to your use case.
+
+## Is there a way to use NTT tokens with chains that don’t currently support NTT?
```
Comment (dawnkelly09) [inline build/contract-integrations/faqs.md:33]:
```suggestion
## Is there a way to use NTT tokens with chains that don't currently support NTT?
```

File: build/contract-integrations/faqs.md [hunk 2]
```diff
@@ -28,4 +28,14 @@ Previously referred to as specialized relayers, custom relayers allow you to bui
...
+
+Yes, NTT tokens can be used with chains that do not support NTT by leveraging the [token bridge](/docs/learn/messaging/token-bridge/){target=\_blank}. For example:
```
Comment (dawnkelly09) [inline build/contract-integrations/faqs.md:35]:
```suggestion
Yes. NTT tokens can be used with chains that do not support NTT by leveraging the [token bridge](/docs/learn/messaging/token-bridge/){target=\_blank}. For example:
```

File: build/contract-integrations/faqs.md [hunk 3]
```diff
@@ -28,4 +28,14 @@ Previously referred to as specialized relayers, custom relayers allow you to bui
...
+
+- **Wrapped token scenario** - a token, such as W token, can be bridged to non-NTT networks using the token bridge. When the token is bridged to a chain like Sui, a wrapped version of the token is created (e.g., Wrapped W token)
```
Comment (dawnkelly09) [inline build/contract-integrations/faqs.md:37]:
```suggestion
- **Wrapped token scenario** - a token, such as the W token, can be bridged to non-NTT networks using the token bridge. When the token is bridged to a chain like Sui, a wrapped version of the token is created (e.g., Wrapped W token)
```

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
Couple little nits. Thanks!

File: build/contract-integrations/faqs.md [hunk 4]
```diff
@@ -28,4 +28,14 @@ Previously referred to as specialized relayers, custom relayers allow you to bui
...
+- **Unwrapping requirement** - tokens bridged using the token bridge cannot be directly transferred to NTT-supported chains. To transfer them, they must first be unwrapped on the non-NTT chain and then transferred via the appropriate mechanism
```
Comment (eshaben) [inline build/contract-integrations/faqs.md:38]:
```suggestion
- **Unwrapping requirement** - tokens bridged using the Token Bridge cannot be directly transferred to NTT-supported chains. To transfer them, they must first be unwrapped on the non-NTT chain and then transferred via the appropriate mechanism
```

File: build/contract-integrations/faqs.md [hunk 5]
```diff
@@ -28,4 +28,14 @@ Previously referred to as specialized relayers, custom relayers allow you to bui
...
+- **Messaging consistency** - the token bridge exclusively uses Wormhole messaging, ensuring consistent communication across all chains, whether or not they support NTT
```
Comment (eshaben) [inline build/contract-integrations/faqs.md:39]:
```suggestion
- **Messaging consistency** - the Token Bridge exclusively uses Wormhole messaging, ensuring consistent communication across all chains, whether or not they support NTT
```

**Review by dawnkelly09 — APPROVED** [review]
nice work!

**Pull request merged**